xc7/fasm2bels/bram_models.py

```diff
@@ -1060,17 +1060,20 @@ def process_bram(conn, top, tile, features):
     assert 'RAMB36.EN_ECC_WRITE' not in tile_features
 
     num_brams = 0
+    num_sdp_brams = 0
     for bram in brams:
         if 'IN_USE' in bram_features[bram]:
             num_brams += 1
+        if 'SDP_READ_WIDTH_36' in bram_features[bram]:
+            num_sdp_brams += 1
```

Comment:
This implementation is wrong?  You documented in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1357#issuecomment-594605888 how to detect how to detect whether a BRAM tile is two BRAM18 SDP's or one BRAM36 SDP's?  Use that?

Reply:
Sure, previous implementation was wrong as it avoided instantiating the RAMB36E in case VPR actually used RAMB36E1. Now it should take this into account. It is not testable yet as RAMB36E1 cannot be used yet.

Reply:
> as RAMB36E1 cannot be used yet.

Yosys cannot infer the RAMB36E1, but you can explicitly instance one for testing purposes